R/run_example.R

```diff
@@ -1,457 +1,458 @@
 #' Example Function
 #'
-#' This function runs a shiny example using different parts of the package
+#' This function runs a shiny example showcasing different parts of the package
+#'
 #' @return a shiny app with examples in
 #' @keywords example
 #' @export
 #' @examples
 #' if (interactive()) {
-#' run_example()
-#'}
-run_example <- function(){
+#'   run_example()
+#' }
+run_example <- function() {
   Months <- c("January", "February", "March")
   Bikes <- c(85, 75, 165)
   Cars <- c(95, 55, 125)
   example_data <- data.frame(Months, Bikes, Cars)
-  tabs <- c(rep("Past Day", 3),
-            rep("Past Week", 3),
-            rep("Past Month", 3),
-            rep("Past Year", 3))
-  Case_manager <- rep(c("David Francis", "Paul Farmer", "Rita Patel"),4)
+  tabs <- c(
+    rep("Past Day", 3),
+    rep("Past Week", 3),
+    rep("Past Month", 3),
+    rep("Past Year", 3)
+  )
+  Case_manager <- rep(c("David Francis", "Paul Farmer", "Rita Patel"), 4)
   Cases_open <- c(3, 1, 2, 24, 16, 24, 98, 122, 126, 1380, 1129, 1539)
   Cases_closed <- c(0, 0, 0, 18, 20, 27, 95, 131, 142, 1472, 1083, 1265)
   data <- data.frame(tabs, Case_manager, Cases_open, Cases_closed)
 
   shiny::shinyApp(
     ui = shiny::fluidPage(
-      title="ShinyGovstyle",
-      cookieBanner("Run Example"),
+      title = "ShinyGovstyle",
+      cookieBanner("shinyGovstyle showcase"),
       skip_to_main(),
-      header("MOJ", "ShinyGovstyle Example",
-             logo="shinyGovstyle/images/moj_logo-1.png", logo_width = 66),
+      header("MoJ", "shinyGovstyle showcase",
+        logo = "shinyGovstyle/images/moj_logo-1.png", logo_width = 66
+      ),
       banner(
         "banner",
         "Beta",
-        'This is a new service \u002D your <a class="govuk-link" href="#">
-        feedback</a> will help us to improve it.'),
+        'This is a new service \u002D your <a class="govuk-link" href="https://github.com/dfe-analytical-services/shinyGovstyle/issues/new/choose">
```

Comment:
could use external link function here once we have it merged in

Reply:
Definitely!